automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala

```diff
@@ -21,26 +21,31 @@ class FreeTrialSpec extends FreeSpec with BeforeAndAfterEach with Matchers with
 
   val adminUser: Credentials = UserPool.chooseAdmin
   implicit val authToken: AuthToken = adminUser.makeAuthToken()
+  val trialKVPKeys = Seq("trialState", "trialBillingProjectName")
 
   var testUser: Credentials = _
   var userAuthToken: AuthToken = _
   var subjectId : String = _
 
-
   override def beforeEach {
     testUser = UserPool.chooseStudent
     userAuthToken = testUser.makeAuthToken()
     subjectId = Orchestration.profile.getUser()(userAuthToken)("userId").toString
-    Try(Thurloe.keyValuePairs.delete(subjectId, "trialState"))
+    Try(trialKVPKeys foreach { k => Thurloe.keyValuePairs.delete(subjectId, k)})
```

Comment:
I wonder if we should delete all `trialState*` keys to be safe.

Reply:
I think I tried that back in November and hit some errors, but will give it another shot now that this is further along.